### PR TITLE
feat: add storage and backup config

### PR DESCRIPTION
**Pull request by antleblanc**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/netapp`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-7267
| License          | BSD 3-Clause

### :sparkles: Features

14b1b7f - feat: add storage and backup config

### :link: Related

- [x] service#45
- [x] https://github.com/ovh/ovh-ui-kit/pull/725
- [x] TRANS-37162 

Signed-off-by: Antoine Leblanc <antoine.leblanc@ovhcloud.com>

cc @marie-j @JeremyDec

### CODE REVIEW

File: packages/manager/modules/server-sidebar/src/sidebar.constants.js [hunk 1]
```diff
@@ -369,6 +369,33 @@ export const DEDICATED_NETWORK_CONFIG = {
   ],
 };
 
+export const STORAGE_BACKUP_CONFIG = {
+  id: 'storage_backup',
+  forceDisplaySearch: true,
+  app: [DEDICATED],
+  feature: 'netapp',
+  icon: 'ovh-font ovh-font-cloudnas',
```
Comment (antleblanc) [inline packages/manager/modules/server-sidebar/src/sidebar.constants.js:377]: Waiting confirmation for icons (here same icon is used three time in a row).

File: packages/manager/modules/server-sidebar/src/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -38,6 +38,8 @@
   "server_sidebar_order_item_vps_title": "Serveurs privés virtuels",
   "server_sidebar_item_web_paas_title": "Web PaaS",
   "server_sidebar_item_web_paas_all_accounts_title": "Tous les projets",
+  "server_sidebar_item_storage_backup_title": "Storage & Backup",
```
Comment (antleblanc) [inline packages/manager/modules/server-sidebar/src/translations/Messages_fr_FR.json:41]: Waiting content confirmation for this entry.

**Review by cbourgois — APPROVED** [review]
Should wait confirmations but 👍  for me.

**Review by JeremyDec — APPROVED** [review]

File: packages/manager/modules/server-sidebar/src/sidebar.constants.js (hunk 1 above)
Comment (marie-j) [reply packages/manager/modules/server-sidebar/src/sidebar.constants.js:377]: PR to add icon: https://github.com/ovh/ovh-ui-kit/pull/725

**Comment by marie-j** [comment]
Commits can be squashed at merge time

**Comment by marie-j** [comment]
Rebase on base branch and squashed commits

**Pull request merged**